hpcbench/benchmark/ior.py

```diff
@@ -217,6 +219,15 @@ def apis(self):
         """List of API to test"""
         return self.attributes['apis']
 
+    def pre_execute(self, execution):
+        """Make sure the named directory is created if possible"""
+        if self.path:
+            if not osp.exists(self.path):
+                os.mkdir(self.path)
+            else:
+                if not osp.isdir(self.path):
```

Comment:
path could be a symbolic link to a directory?
What about checking that `os.path.realpath(self.path)` is a directory instead ?